### PR TITLE
Valkey 8 GA blog

### PR DESCRIPTION
**Pull request by stockholmux**
### Description

Adds a blog post for Valkey 8 being GA. To remain in draft status until GA is ready and after the downloads page is ready.
 
### Issues Resolved

n/a

### Check List
- [x] Commits are signed per the DCO using `--signoff`

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: content/blog/2024-09-16-valkey-8-ga.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
++++
+title=  "Generally Available: Valkey 8.0.0"
+description = "Today marks a milestone for the Valkey project: the first major release."
+date= 2024-09-16 01:01:01
+authors= ["kyledvs"]
++++
+
+The first ever release of Valkey, 7.2.5, became generally available more than 5 months ago.
+While the initial release was milestone, it focused on compatibility and license continuity; bringing no new features to the table.
```
Comment (madolson) [inline content/blog/2024-09-16-valkey-8-ga.md:9]:
```suggestion
While the initial release was a milestone, it focused on compatibility and license continuity; bringing no new features to the table.
```

File: content/blog/2024-09-16-valkey-8-ga.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+Today marks a different milestone for the Valkey project: the first major release.
+Valkey 8.0.0 continues the the traditions of the seven major versions of Redis that precede it by bringing improvements to speed and efficiency alongside new features.
+
+Key properties of the Valkey project are transparency and collaboration.
+As a consequence of Valkey 8.0.0 being developed entirely in the open, the team has already written about both the big and small features of the release.
+The best overview is the RC1 blog which breaks down all the changes and features in the release into a few sections: [performance](/blog/valkey-8-0-0-rc1/#performance), [reliability](/blog/valkey-8-0-0-rc1/#reliability), [replication](/blog/valkey-8-0-0-rc1/#replication), [observability](https://valkey.io/blog/valkey-8-0-0-rc1/#observability), and [efficiency](/blog/valkey-8-0-0-rc1/#efficiency).
+Additionally, there are deep dives on the [speed](/blog/unlock-one-million-rps/) and [efficiency](/blog/valkey-memory-efficiency-8-0/) improvements in Valkey 8.0.0.
```
Comment (madolson) [inline content/blog/2024-09-16-valkey-8-ga.md:16]: We'll maybe want to link part 2 of one million rps as well.

File: content/blog/2024-09-16-valkey-8-ga.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+While this is a major version, Valkey takes command set compatibility seriously: Valkey 8.0.0 makes no backwards incompatible changes to the existing commands or their responses.
```
[the remark this answers is not in the document]
Comment (stockholmux) [reply content/blog/2024-09-16-valkey-8-ga.md:18]: So, would you suggest changing the statement or just adding a link to release notes?

[the remark this answers is not in the document]
Comment (madolson) [reply content/blog/2024-09-16-valkey-8-ga.md:18]: I would adjust it to say there are minor behavior changes to undefined behavior, I would add a link to the release notes to say to review it.

File: content/blog/2024-09-16-valkey-8-ga.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+title=  "Generally Available: Valkey 8.0.0"
+description = "Today marks a milestone for the Valkey project: the first major release."
+date= 2024-09-16 01:01:01
+authors= ["kyledvs"]
++++
+
+The first ever release of Valkey, 7.2.5, became generally available more than 5 months ago.
+While the initial release was a milestone, it focused on compatibility and license continuity; bringing no new features to the table.
+Today marks a different milestone for the Valkey project: the first major release.
+Valkey 8.0.0 continues the the traditions of the seven major versions of Redis that precede it by bringing improvements to speed and efficiency alongside new features.
```
Comment (madolson) [inline content/blog/2024-09-16-valkey-8-ga.md:11]:
```suggestion
Valkey 8.0.0 continues the traditions of the seven major versions of Redis that precede it by bringing improvements to speed and efficiency alongside new features.
```

Comment (stockholmux) [reply content/blog/2024-09-16-valkey-8-ga.md]: Thanks. I've read this thing 20 times and I didn't didn't see see this this.

**Review by madolson — APPROVED** [review]

**Pull request merged**